ui-v2/app/components/healthcheck-status.js

```diff
@@ -1,5 +1,11 @@
 import Component from '@ember/component';
-
+import { subscribe } from 'consul-ui/utils/computed/purify';
 export default Component.extend({
-  classNames: ['healthcheck-status'],
+  tagName: '',
+  count: subscribe('value', function(value) {
```

Comment:
I think you could use `alias` here too. https://www.emberjs.com/api/ember/3.7/functions/@ember%2Fobject%2Fcomputed/alias